script/apply-patches

```diff
@@ -37,7 +39,8 @@ def apply_patches_for_dir(directory):
 
     (success, failed_patches) = apply_patches_from_directory(repo, root)
     if not success:
-      return '{0} failed to apply'.format(os.path.basename(failed_patches[0]))
+      patch_path = failed_patches[0].get_file_path()
+      return '{0} failed to apply'.format(os.path.basename(patch_path))
```

Comment:
I don't understand the intent of this change.

What's the purpose of `get_file_path()` here when the path is stripped back out on the next line?

Reply:
Before this PR `failed_patches[0]` was a path to a patch file (a string), but now it's a `Patch` instance. To get its path I had to call `.get_file_path()`.